src/pupil_labs/neon_recording/stream/gaze_stream.py

```diff
@@ -37,13 +38,7 @@ def keys(self):
 class GazeArray(Array[GazeRecord], GazeProps):
```

Comment:
Since sampling is eternalized, I removed the Stream/SampledStream classes from the hierarchy. `rec.gaze` now returns a GazeArray, which is just a structured array, directly, which seems clean to me.

Reply:
The original plnr implementation supported chaining of sample(), does that still work with this change?

ie:

```python
rec.gaze.sample(rec.scene.ts).sample(np.arange(0, 10e9, 1e9/40)
```